From d24f741a17411a271ac77ae8fcca256f278fa8eb Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 11 Jun 2024 16:01:30 +0000
Subject: [PATCH] IWYU: missing include for usage of SharedURLLoaderFactory in
 headless_field_trials.cc

Bug: 41455655
Change-Id: Ifdb35ffc5aacf2a41ff2035ec270a6872e887816
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5616570
Commit-Queue: Andrey Kosyakov <caseq@chromium.org>
Reviewed-by: Andrey Kosyakov <caseq@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1313446}
---
 headless/lib/browser/headless_field_trials.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/headless/lib/browser/headless_field_trials.cc b/headless/lib/browser/headless_field_trials.cc
index e02b6f448d07d..9156c999ddd11 100644
--- a/headless/lib/browser/headless_field_trials.cc
+++ b/headless/lib/browser/headless_field_trials.cc
@@ -23,6 +23,7 @@
 #include "components/variations/variations_seed_store.h"
 #include "components/variations/variations_switches.h"
 #include "content/public/common/content_switch_dependent_feature_overrides.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
 
 namespace headless {
 
